app/cronjob/trailingTradeHelper/configuration.js

```diff
@@ -202,6 +202,74 @@ const getMaxPurchaseAmount = async (
 
```

Comment:
Again, I see lots of formatting issue.
Use vscode and it should fix your formatting issue when you save it.

![image](https://user-images.githubusercontent.com/5715919/120090399-8dbfe600-c145-11eb-9f3b-b5ad3723f651.png)